doc/user/logging.md

```diff
@@ -60,6 +60,53 @@ $ go run main.go --zap-level=1
 {"level":"info","ts":1559866310.065119,"logger":"scoped","msg":"Printing at INFO level"}
 {"level":"debug","ts":1559866310.065123,"logger":"scoped","msg":"Printing at DEBUG level"}
 ```
+## Custom zap logger
+
+In order to use a custom zap logger, [`zap`][controller_runtime_zap] from controller-runtime can be utilized to wrap it in a logr implementation.
+
+Below is an example illustrating the use of [`zap-logfmt`][logfmt_repo] in logging.
+
+### Example
+
+```Go
+package main
+
+import (
+	"github.com/go-logr/zapr"
+	zaplogfmt "github.com/sykesm/zap-logfmt"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	zapcr "sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var log logr.Logger
+
+func main() {
+	configLog := zap.NewProductionEncoderConfig()
+	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
+		encoder.AppendString(ts.UTC().Format(time.RFC3339))
+	}
+	logfmtEncoder := zaplogfmt.NewEncoder(configLog)
+
+	// Construct a new logr.logger.
+	log = zapcr.New(zapcr.UseDevMode(true), zapcr.WriteTo(os.Stdout), zapcr.Encoder(logfmtEncoder))
+
+	// Set the controller logger to log, which will
+	// be propagated through the whole operator, generating
+	// uniform and structured logs.
+	logf.SetLogger(log)
+
+	log.Info("Printing at DEBUG LEVEL")	
+}
+```
+
+#### Output using custom zap logger
+```console
+$ go run main.go
+ts=2020-02-25T22:42:43Z level=info msg="Printing at DEBUG level"
+ts=2020-02-25T22:42:43Z level=info msg="Printing at DEBUG level"
```

Comment:
The output needs have been obtained from the Memcached Go Sample with these changes. 
We need to show how the logs in the operator project container will be. 
See: https://github.com/operator-framework/operator-sdk-samples/tree/master/go/memcached-operator